core/templates/pages/admin-page/admin-blog-admin-common.module.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the common code between admin pages.
+ */
+
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RolesAndActionsVisualizerComponent } from './roles-tab/roles-and-actions-visualizer.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+  ],
+  declarations: [
+    RolesAndActionsVisualizerComponent
+  ],
+  entryComponents: [
+    RolesAndActionsVisualizerComponent
+  ],
+  exports: [
+    RolesAndActionsVisualizerComponent
+  ],
+})
+
+export class AdminBlogAdminCommonModule { }
```

Comment:
Here and all other modules created, contains components included in multiple modules. This causes angular build to fail.